src/modules/spaces/services/spaces/spaces.service.ts

```diff
@@ -0,0 +1,10 @@
+import { ReadingRoomInfo } from '@reading-room/types';
+import { ApiService } from '@shared/services/api-service';
+
+import { SPACES_SERVICE_BASE_URL } from './spaces.service.const';
+
+export class SpacesService {
```

Comment:
De spaces queries zijn reeds gedefinieerd in `reading-room-service`. Ik zou ofwel de `getById` daaruit gebruiken ofwel die service renamen.

(die `getById` heeft wel nog geen `/spaces/` in de query

Reply:
Fixed in https://github.com/viaacode/hetarchief-client/pull/217/commits/869c51f640708ebf4b7f189f5cf76b9d33b8b330, `getById` verdwijnt en wordt vervangen door `getBySlug`